README.md

```diff
@@ -14,6 +14,57 @@ This SDK current supports the following versions of CloudEvents:
 
 Package **cloudevents** provides primitives to work with CloudEvents specification: https://github.com/cloudevents/spec.
 
+Creating minimal CloudEvents with data in version 1.0.0:
```

Comment:
Just say `Sending CloudEvents:`

We can see the version and data in the example and should assume 1.0 anyways.